__tests__/app.test.js

```diff
@@ -206,4 +206,45 @@ describe('/api/articles/:article_id/comments', () => {
             expect(message).toBe('Bad Request')
         })
     })
+})
+
+describe('/api/articles/:article_id', () => {
+    test('PATCH 202: Should update an article by ID and return the updated article', () => {
+        const updArticle = {
+            inc_votes: 10
+        }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(updArticle)
+        .expect(202)
+        .then(({ body: { article }}) => {
+            expect(article.article_id).toBe(1)
+            expect(article.title).toBe("Living in the shadow of a great man")
+            expect(article.votes).toBe(110)
+        })
```

Comment:
Great happy path test, would be good to also check it's working for a negative `inc_vote` value 🔍